package-lock.json

```diff
@@ -30,7 +30,7 @@
                 "eslint": "^8.50.0",
                 "eslint-config-google": "^0.14.0",
                 "fs-extra": "^11.1.1",
-                "glob": "^10.3.5",
+                "glob": "^10.3.7",
```

Comment:
For some reason from now on dependency with types is required, I wonder why it wasn't the case earlier.

https://www.npmjs.com/package/@types/glob